gitdb/pack.py

```diff
@@ -247,6 +247,15 @@ def write(self, pack_sha, write):
         return sha
 
 
+def _frombytes(a, s):
```

Comment:
From GitPython 3.0, python 2 is no longer supported. Thus it would be alright to call the function directly. When done, I would create a new major release to prevent breakage.